===================================================================
RCS file: /opt/sourcecast/data/ccvs/repository/rome/build.xml,v
retrieving revision 1.14
retrieving revision 1.15
diff -u -r1.14 -r1.15
--- rome/build.xml	2006/12/04 21:47:37	1.14
+++ rome/build.xml	2006/12/26 14:35:46	1.15
@@ -131,6 +131,9 @@
       <fileset dir="src/data">
         <include name="**/*.xml" />
       </fileset>
+      <fileset dir="src/test">
+        <include name="**/*.properties" />
+      </fileset>
     </copy>    
   </target>
   
